theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -102,45 +211,134 @@ const CurveEditorPopover: React.FC<
             ],
           })
         })
+
+        props.onRequestClose()
       },
     }
-  }, [props.layoutP, props.index])
+  }, [props.layoutP, props.index, presetResults])
 
   const inputRef = useRef<HTMLInputElement>(null)
   useLayoutEffect(() => {
     inputRef.current!.focus()
-    inputRef.current!.setSelectionRange(0, 100)
   }, [])
 
   const {index, trackData} = props
   const cur = trackData.keyframes[index]
   const next = trackData.keyframes[index + 1]
 
-  const cssCubicBezierString = keyframesToCssCubicBezierArgs(cur, next)
+  // Need some padding *inside* the SVG so that the handles and overshoots are not clipped
+  const svgPadding = 0.12
+  const svgCircleRadius = 0.08
+  const svgColor = '#b98b08'
+
+  // query
+  const useQuery = /^[A-Za-z]/.test(filter)
+  const optionsEmpty = useQuery && presetResults.length === 0
 
   return (
     <Container>
-      <Label>cubic-bezier{'('}</Label>
-      <BasicStringInput
-        value={cssCubicBezierString}
-        {...fns}
-        isValid={isValid}
-        inputRef={inputRef}
-        onBlur={props.onRequestClose}
-      />
-      <Label>{')'}</Label>
+      <InputContainer>
+        <SearchBox
+          value={filter}
+          placeholder="Search presets..."
+          onChange={(e) => {
+            setFilter(e.target.value)
+          }}
+          ref={inputRef}
+          onBlur={() => props.onRequestClose()}
+          onKeyPress={(e) => {
+            if (e.key === 'Escape') {
+              props.onRequestClose()
+            } else if (e.key === 'Enter') {
```

Comment:
This is yet to be implemented, right?

Here I'm entering "l" for "linear" and hitting Enter, but linear isn't applied here:
https://user-images.githubusercontent.com/593118/145195837-98bbdc28-aacf-4949-8ad5-b78d48f1f905.mp4

Reply:
Probably an accidental omission, will fix